### PR TITLE
add Card component

### PR DESCRIPTION
**Pull request by turfle**

### CODE REVIEW

**Review by mtseluiko — APPROVED** [review]

File: frontend/src/components/Card/CardWithLink.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { FC, ComponentProps } from 'react';
+import Link from '../Link';
+import Card from '.';
+
+type Props = ComponentProps<typeof Card> & {
+  to: string,
+}
+
+const CardComponent: FC<Props> = ({
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Card/CardWithLink.tsx]: rename CardComponent  to CardWitLink

File: frontend/src/components/Card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <div className={classes.title}>{name}</div>
+      <div className={classes.description}>{description}</div>
+
+      <div className={classes.tags}>
+        {tags.map((tag => <Tag text={tag} />))}
+      </div>
+
+      <ProgressBar goal={goal} percent={percent} />
+    </div>
+  </article>
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Card/index.tsx]: This seems to be the ProjectCard.

Let's create 2 separate components: Card and ProjectCard.
The card will be a common component with basic styles, and the ProjectCard will use the card and pass content there as a child.

Even you can leave card-image inside the card and use if image property defined.

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/components/Card/CardWithLink.tsx (hunk 1 above)
Comment (turfle) [reply frontend/src/components/Card/CardWithLink.tsx]: done

File: frontend/src/components/Card/index.tsx (hunk 2 above)
Comment (turfle) [reply frontend/src/components/Card/index.tsx]: done

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**